notebooks/community/gapic/custom/showcase_custom_image_classification_online_explain_example_based_api.ipynb

```diff
@@ -0,0 +1,3291 @@
+{
```

Comment:
<p>Generally, we don't pin the notebook versions, so that we catch errors with new versions.</p><p> </p><p> If you want to keep the pins, I might suggest keeping one flow for our CI tests, like this:</p><p> </p><p>if os.getenv("IS_TESTING"):</p><p>	// unpinned pip installs</p><p>else:</p><p>	//pinned installs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/716/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Makes sense Karl. I'll go with if-else approach. Thanks</p>

Reply:
<p>done</p>